### ISSUE TITLE
Switch to fullscreen intermittently fails to render.

### ISSUE THREAD

**Issue by jhenzie** [alacritty/alacritty#6485]
Switcing between windowed and fullscreen (Alt - Enter) intermittently leaves alaccrity with no output until it receives input (keypress).

6 cycles between windowed and fullscreen are usually sufficient to trigger the bug.

Unable to capture screen output but it has the appearance of a black screen which would suggest the framebuffer is not being rendered at all since the color scheme is ignored.

Pressing alphanumeric keys will resolve the problem. Pressing backspace, when there is nothing to delete, does not resolve the problem.

### System

OS: Windows
Version: alacritty 0.11.0

**Comment by chrisduerr** [comment]
This seems similar to https://github.com/alacritty/alacritty/issues/6472.

Does minimize also clear the window for you?

**Comment by jhenzie** [comment]
No, returning to windowed mode results in a normally rendered terminal.

**Comment by chrisduerr** [comment]
Can you try the latest master please? Also which Windows version are you running?

**Comment by jhenzie** [comment]
Windows 11 22H2 Build 22621.819

**Comment by jhenzie** [comment]
Still occurs with master -- version reported as alacritty 0.12.0-dev (2d619850)

**Comment by chrisduerr** [comment]
Can you give me your `alacritty -v` output?

**Comment by jhenzie** [comment]
Installed version
PS [C:] [..\Justin Henzie] > alacritty.exe -v
PS [C:] [..\Justin Henzie] > Created log file at "C:\Users\JUSTIN~1\AppData\Local\Temp\Alacritty-2972.log"
[0.000390700s] [INFO ] [alacritty] Welcome to Alacritty
[0.001513200s] [INFO ] [alacritty] Version 0.11.0
[0.010507100s] [INFO ] [alacritty] Configuration files loaded from: "C:\\Users\\Justin Henzie\\AppData\\Roaming\\alacritty\\alacritty.yml"
[0.127840000s] [INFO ] [alacritty] Running on Intel(R) UHD Graphics 630
[0.128290200s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.161555700s] [INFO ] [alacritty] Display scale factor: 1.25
[0.215668600s] [INFO ] [alacritty] Cell size: 11 x 23
[0.216017600s] [INFO ] [alacritty] Padding: 0 x 0
[0.216353100s] [INFO ] [alacritty] Width: 800, Height: 600
[0.255064800s] [INFO ] [alacritty] PTY dimensions: 26 x 72
[0.261397900s] [INFO ] [alacritty] Initialisation complete
[0.263270900s] [INFO ] [alacritty] Padding: 0 x 0
[0.263580200s] [INFO ] [alacritty] Width: 800, Height: 600

Built version from master
PS [D:] [..\release] > ./alacritty.exe -v
PS [D:] [..\release] > Created log file at "C:\Users\JUSTIN~1\AppData\Local\Temp\Alacritty-21900.log"
[0.000013200s] [INFO ] [alacritty] Welcome to Alacritty
[0.000917700s] [INFO ] [alacritty] Version 0.12.0-dev (2d619850)
[0.005213200s] [INFO ] [alacritty] Configuration files loaded from: "C:\\Users\\Justin Henzie\\AppData\\Roaming\\alacritty\\alacritty.yml"
[0.024870700s] [INFO ] [alacritty] Window scale factor: 1.25
[0.075009900s] [INFO ] [alacritty] Using WGL
[0.096360100s] [INFO ] [alacritty] Running on Intel(R) UHD Graphics 630
[0.096765700s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.201089800s] [INFO ] [alacritty] Cell size: 11 x 23
[0.201421800s] [INFO ] [alacritty] Padding: 0 x 0
[0.202315800s] [INFO ] [alacritty] Width: 800, Height: 600
[0.237927800s] [INFO ] [alacritty] PTY dimensions: 26 x 72
[0.244796700s] [INFO ] [alacritty] Initialisation complete

**Comment by chrisduerr** [comment]
With the version built from master, could you execute it with the following options?

```
alacritty -v -o "debug.renderer=gles2_pure"
```

That should change the output to contain "Using OpenGL ES 2.0 renderer" instead of "Using OpenGL 3.3 renderer" if it worked correctly.

Then you can try reproducing that issue again in the same terminal and let me know if it still shows up.

**Comment by jhenzie** [comment]
Terminal is non functional after executing alacritty -v -o "debug.renderer=gles2_pure"

![image](https://user-images.githubusercontent.com/21959/204796294-e616b0d0-f9fb-49bb-a0d3-ef5a06e16b7d.png)

**Comment by chrisduerr** [comment]
Can you try just `gles2`, without `_pure`?

**Comment by jhenzie** [comment]
\alacritty.exe -v -o "debug.renderer=gles2"
PS [D:] [..\release] > Created log file at "C:\Users\JUSTIN~1\AppData\Local\Temp\Alacritty-12108.log"
[0.000007900s] [INFO ] [alacritty] Welcome to Alacritty
[0.000824600s] [INFO ] [alacritty] Version 0.12.0-dev (2d619850)
[0.009657300s] [INFO ] [alacritty] Configuration files loaded from:
                                     "C:\\Users\\Justin Henzie\\AppData\\Roaming\\alacritty\\alacritty.yml"
[0.025228800s] [INFO ] [alacritty] Window scale factor: 1.25
[0.087972000s] [INFO ] [alacritty] Using WGL
[0.110819100s] [INFO ] [alacritty] Running on Intel(R) UHD Graphics 630
[0.111048900s] [INFO ] [alacritty] Using OpenGL ES 2.0 renderer
[0.116909100s] [INFO ] [alacritty] Using dual source blending
Error: There was an error initializing the shaders: Failed compiling shader: ERROR: 1:4: '#extension' :  'GL_EXT_blend_func_extended' is not supported
ERROR: 1:45: 'gl_SecondaryFragColorEXT' : undeclared identifier
ERROR: 1:45: 'assign' :  cannot convert from 'const 4-component vector of float' to 'float'
ERROR: 1:59: 'gl_SecondaryFragColorEXT' : undeclared identifier
ERROR: 1:59: 'assign' :  cannot convert from '4-component vector of mediump float' to 'mediump float'
ERROR: 1:70: 'gl_SecondaryFragColorEXT' : undeclared identifier
ERROR: 1:70: 'assign' :  cannot convert from '4-component vector of mediump float' to 'mediump float'

**Comment by kchibisov** [comment]
So I guess their issue with `gles2` renderer is that they have only ARB extension. And with `gles2_pure` that their driver is not compliant because it doesn't allow RGB in RGBA texture at the same time, while it should do so (We ask for OpenGL 2.1 which should have that).

If something fails to render in the full-screen it might be some driver issue to be fair, though we don't use exclusive full-screen so that sorts of surprising.

**Comment by chrisduerr** [comment]
Oh yeah, might be interesting to see if switching between maximize and not maximized causes similar issues? Maybe it's just the rapid resizing?
